packages/js/extend-cart-checkout-block/$slug.php.mustache

```diff
@@ -18,34 +18,119 @@
  *
  * @package         {{namespace}}
  */
-add_action('woocommerce_blocks_loaded', function() {
-    require_once __DIR__ . '/{{slug}}-blocks-integration.php';
-	add_action(
-		'woocommerce_blocks_cart_block_registration',
-		function( $integration_registry ) {
-			$integration_registry->register( new {{slugPascalCase}}_Blocks_Integration() );
-		}
-	);
-	add_action(
-		'woocommerce_blocks_checkout_block_registration',
-		function( $integration_registry ) {
-			$integration_registry->register( new {{slugPascalCase}}_Blocks_Integration() );
-		}
-	);
-});
+add_action( 'woocommerce_blocks_loaded',
```

Comment:
We should move to using `woocommerce_init` as it's the safest and run in both themes and plugins.

Reply:
Created a follow up here
https://github.com/woocommerce/woocommerce/issues/48359